messaging/src/main/java/org/apache/falcon/messaging/JMSMessageConsumer.java

```diff
@@ -92,7 +92,8 @@ public void startSubscriber() throws FalconException {
 
             topicSession = (TopicSession) connection.createSession(false, Session.AUTO_ACKNOWLEDGE);
             Topic destination = topicSession.createTopic(topicName);
-            topicSubscriber = topicSession.createDurableSubscriber(destination, FALCON_CLIENT_ID);
+            topicSubscriber = topicSession.createDurableSubscriber(destination, FALCON_CLIENT_ID,
```

Comment:
@pallavi-rao post-processing now sends only user notifications (the code in postprocessing to send system notification to FALCON.ENTITY.TOPIC is removed since Oozie JMS covers that). In that regard, I think it is a backward compatible  issue -- Users need to enable Oozie JMS notification. This is anyways needed for Instance Search to work.

Regarding the second one, are you referring to the TextMessage and MapMessage cases in the code? MapMessage in the JMS Consumer reads messages from post processing while TextMessage represents the Oozie JMS messages. Can you please confirm?